helm/reana/values.yaml

```diff
@@ -11,6 +11,11 @@ debug:
 eos:
   enabled: false
 
+workspaces:
+  paths:
+    - /tmp/reana
+    - /tmp/foo
```

Comment:
This is nice for testing, but perhaps should be removed in the final PR version?

Reply:
Oops, leftovers :sweat_smile: